jsonforms-editor/src/JsonFormsEditor.tsx

```diff
@@ -142,6 +148,7 @@ export const JsonFormsEditor: React.FC<JsonFormsEditorProps> = ({
           propertyRenderers={propertyRenderers}
           header={headerComponent}
           footer={footerComponent}
+          paletteTabs={paletteTabs ?? undefined}
```

Comment:
paletteTabs is always defined, isn't it because of the default value in the constructor?

Reply:
Yes, but they can be `null` (when no tabs are desired). The check here is for that case.

Reply:
hmm, I see, shouldn't we remove the default from the constructor then?

Reply:
The default kicks in when no `paletteTabs` are passed. Only when `null` is explicitly passed in we show no additional tabs. It's also consistent with the other customizable parts of the editor: editor tabs (preview), header & footer. It would make sense to keep the defaults in the constructor. What do you think?

Reply:
I see your point, can we type this in the constructor accordingly? or maybe use a symbol to indicate this case explicitly?

Reply:
I explicitly added `null` to the types (did it for the other params as well, e.g. `editorTabs`)

Reply:
You tested that passing null is correctly handled in the constructor?